mappings/net/minecraft/client/render/entity/feature/StuckObjectsFeatureRenderer.mapping

```diff
@@ -4,6 +4,11 @@ CLASS net/minecraft/class_4507 net/minecraft/client/render/entity/feature/StuckO
 	METHOD method_22130 renderObject (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_1297;FFFF)V
 		ARG 1 matrix
 		ARG 2 vertexConsumers
-		ARG 5 tickDelta
+		ARG 3 light
+		ARG 4 entity
+		ARG 5 dirX
+		ARG 6 dirY
+		ARG 7 dirZ
```

Comment:
Dir?

Reply:
direction i supposed, from the old pr not changed